bin/pftt2_release.bat

```diff
@@ -0,0 +1,41 @@
+@ECHO OFF
+
+REM set important env vars
+IF DEFINED PFTT_SHELL GOTO :skip_set_env
+CALL %~dp0set_env.cmd
+:skip_set_env
+
+SET PFTT_LIB=%PFTT_HOME%\lib
+SET PFTT_BUILD=%PFTT_HOME%\build
+
+REM Check if build directory exists
+if exist %PFTT_BUILD% (
+	cd ..
```

Comment:
So this still expects to run the script one needs to switch to `%FTT_BIN%`? How about `cd %PFTT_HOME%`? Or `pushd %PFTT_HOME%`. And then at the bottom no `cd %PFTT_BIN%`. Please see the the first item in the list of my ask :)

Thanks.

Reply:
Updated once more. Instead of switching directories, I found it easier to just do every from where the batch file is executed. Is it fine doing it this way?

Reply:
Actually, I just noticed. The only problem now is when it tries to call set_env.cmd. If it's not in bin, then it won't find the file. Should I update it so that it doesn't depend on set_env.cmd?